src/main/java/twilightforest/block/BlockTFCicada.java

```diff
@@ -11,11 +16,64 @@ public class BlockTFCicada extends BlockTFCritter {
 
     protected BlockTFCicada() {
         super();
+        this.dropItem = Items.dye;
+        this.dropMeta = 2;
     }
 
     @Override
     public TileEntity createTileEntity(World world, int metadata) {
         return new TileEntityTFCicada();
     }
 
+    /**
+     * Triggered whenever an entity collides with this block (enters into the block). Args: world, x, y, z, entity
+     */
+    @Override
+    public void onEntityCollidedWithBlock(World world, int x, int y, int z, Entity entity) {
+        super.onEntityCollidedWithBlock(world, x, y, z, entity);
+        // TwilightForestMod.proxy.stopSound(worldIn, x, y, z);
+    }
+
+    @Override
+    public boolean dropCritterIfCantStay(World world, int x, int y, int z) {
+        if (super.dropCritterIfCantStay(world, x, y, z)) {
```

Comment:
Super returns false whenever the block it's on is destroyed. Is it intended that the sound continues to play in that scenario?

Reply:
Nope
Totally not
Thanks for pointing at this